collect_app/src/main/java/org/odk/collect/android/audio/AudioRecordingControllerFragment.java

```diff
@@ -53,11 +55,13 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
             } else if (session.getFile() == null) {
                 binding.getRoot().setVisibility(VISIBLE);
 
+                binding.timeCode.setText(LengthFormatterKt.formatLength(session.getDuration()));
+
                 if (session.getPaused()) {
-                    binding.pauseRecording.setText(R.string.resume_recording);
+                    binding.pauseRecording.setIcon(ContextCompat.getDrawable(getContext(), R.drawable.ic_play_arrow_24dp));
```

Comment:
I think this should be a red record circle.

Reply:
Nice touch. I like that!

Reply:
Actually after playing with this it makes it feel like we're recording while paused as there is a big red recording icon. Lets not change this at the moment.

Reply:
I don’t think play is at all clear, though. Does it indicate ongoing playback? Playing back what you just recorded before stopping? Even knowing that it was supposed to be resume I wasn’t sure that’s what it would do. 

What do other recording apps do?  Maybe a black circle? Or maybe the scale of the button is the challenge?